electrum/commands.py

```diff
@@ -592,7 +592,7 @@ async def verifymessage(self, address, signature, message):
 
     @command('wp')
     async def payto(self, destination, amount, fee=None, feerate=None, from_addr=None, from_coins=None, change_addr=None,
-                    nocheck=False, unsigned=False, rbf=None, password=None, locktime=None, wallet: Abstract_Wallet = None):
+                    nocheck=False, unsigned=False, rbf=None, password=None, locktime=None, addtransaction=True, wallet: Abstract_Wallet = None):
```

Comment:
The default should be "False", as "True" would be a significant change in behaviour.

Reply:
Just a question: why exactly is transaction added to wallet after a full network round trip? I can see it not only from CLI, but from GUI. When sending a transaction, it tells tx hash, and only after some time (not significant of course) it is showing "synchronizing (3/3)" and then showing that in GUI + notification.
As to review comment, will change it now

Reply:
It works the way it works due to historical reasons.
There used to be no "local" transaction support; the tx history in the wallet used to always mirror the last connected server.

It could be changed now but the implications should be considered. For example, what if the tx is rejected by the server? (e.g. imagine the user creating a non-standard tx, rejected by bitcoind policy, e.g. creating dust outputs - I suspect that sort of scenario might result in increased support load).

Reply:
Hmm, I am unsure but if broadcast fails then no synchronizing step is done. This fix seems to be more of a workaround than a real solution for me, but it works.